src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
+
+type Color = 'red' | 'green' | 'blue';
+type Snape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Better to use `enum` here, you will be able to write `shape: Shape = Shape.triangle;` instead of `shape: Shape = 'triangle';`
Also, `Snape` -> `Shape`